pkg/apis/configuration/validation/validation.go

```diff
@@ -57,6 +57,20 @@ func validateTLS(tls *v1alpha1.TLS, fieldPath *field.Path) field.ErrorList {
 	return validateSecretName(tls.Secret, fieldPath.Child("secret"))
 }
 
+func validateTime(time string, fieldPath *field.Path) field.ErrorList {
```

Comment:
I wasn't sure, ParseTime has a lot of tests and this `validTime` function pretty much only calls that, I can add a test for consistency with other validation functions, however I think it will be repetition of tests.